pages/index.js

```diff
@@ -1,69 +1,479 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../styles/Home.module.css";
+import styled from "styled-components";
+import Button from "../components/Button";
+import { useRef } from "react";
+import SVG1 from '../public/svg/LandingPage/svg1';
+import SVG2 from '../public/svg/LandingPage/svg2';
+import SVG3 from '../public/svg/LandingPage/svg3';
+import SVG4 from '../public/svg/LandingPage/svg4';
+import SVG5 from '../public/svg/LandingPage/svg5';
+import SVG6 from '../public/svg/LandingPage/svg6';
+import SVG7 from '../public/svg/LandingPage/svg7';
+import SVG8 from '../public/svg/LandingPage/svg8';
+import SVG9 from '../public/svg/LandingPage/svg9';
+
+import Logo from '../public/svg/logo';
+
+//---------styling-starts----------
+
+
+const Navlink = styled.a`
+  margin: 30px;
+  font-family: Oswald;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 30px;
+  line-height: 44px;
+  text-align: center;
+  color: black;
+  cursor: pointer;
+  &:hover {
+    text-decoration: none;
+    color: grey;
+  }
+  &:after {
+    text-decoration: none;
+    /* color:black; */
+  }
+  @media (max-width: 768px) {
+    display: none;
+  }
+`;
+
+const Title = styled.p`
```

Comment:
Please use semantic tags, example `h1` can be used for title, similarly use the `button` tag for buttons.

I think most of these styled components can be move to the `components` folder in their own file.

Checkout: https://primer.style/css/utilities/typography

Reply:
Okay, will do this

Reply:
Styled components which are only significant to index.js, are in index.js. The common ones I have put in components folder.

Reply:
Please checkout: https://primer.style/css/utilities/typography

I think a lot of these *can be* re-used.

Reply:
Will move it.